public/app/plugins/datasource/prometheus/querybuilder/operations.ts

```diff
@@ -80,11 +85,215 @@ export function getOperationDefinitions(): QueryBuilderOperationDef[] {
       renderer: (model, def, innerExpr) => innerExpr,
       addOperationHandler: addNestedQueryHandler,
     },
+    createFunction({ id: PromOperationId.Absent }),
+    createRangeFunction(PromOperationId.AbsentOverTime),
+    createFunction({
+      id: PromOperationId.Acos,
+      category: PromVisualQueryOperationCategory.Trigonometric,
+    }),
+    createFunction({
+      id: PromOperationId.Acosh,
+      category: PromVisualQueryOperationCategory.Trigonometric,
+    }),
+    createFunction({
+      id: PromOperationId.Asin,
+      category: PromVisualQueryOperationCategory.Trigonometric,
+    }),
+    createFunction({
+      id: PromOperationId.Asinh,
+      category: PromVisualQueryOperationCategory.Trigonometric,
+    }),
+    createFunction({
+      id: PromOperationId.Atan,
+      category: PromVisualQueryOperationCategory.Trigonometric,
+    }),
+    createFunction({
+      id: PromOperationId.Atanh,
+      category: PromVisualQueryOperationCategory.Trigonometric,
+    }),
+    createFunction({
+      id: PromOperationId.BottomK,
+      category: PromVisualQueryOperationCategory.Aggregations,
+    }),
```

Comment:
I think this is an aggregation and should be moved here: https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/prometheus/querybuilder/aggregations.ts#L19